2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
On Tue, Mar 24, 2015 at 02:21:07PM -0700, Piotr Banaszkiewicz wrote:
> I'm not able to get rid of jiggling entirely - this may be because I
> have a dataset with non-unique index (aka date).

Right, you want to sort by increasing count (which will automatically
sort by increasing date).  It looks like you currently have multiple
entries for one date and the higher-count entry is currently landing
before the lower count entry.

> To fix horrible jiggling in this one I removed entries with
> duplicated dates leaving the latest entry with biggest number of
> instructors

That works.  I'm not sure how you're creating the plots, but most
plotting packages have something that lets you draw stepped lines.
You should use something like gnuplot's 'steps' [1].  With that
plotting style, it won't matter whether you de-dup the dates (as long
as you've sorted by increasing count).

[1]: http://gnuplot.sourceforge.net/demo/steps.html